library/src/scala/tasty/reflect/Printers.scala

```diff
@@ -1360,6 +1368,9 @@ trait Printers
             case tpe @ Type.SymRef(IsClassDefSymbol(sym), _) if sym.name.endsWith("$") =>
               printType(tpe)
               this += ".type"
+            case tpe @ Type.SymRef(sym, _) if sym.isTerm =>
```

Comment:
Now you can use `case tpe @ Type.SymRef(IsTermSymbol(sym), _) =>`

Reply:
Now we implemented `isTerm` as a helper method.